src/controller/WebController.js

```diff
@@ -0,0 +1,173 @@
+    }
+  };
+
+  validateBuyMoney = (buyMoney) => {
+    Validator.isNumber(buyMoney);
+    Validator.isDividedByThousand(buyMoney);
+    Validator.isPositiveInteger(buyMoney);
+  };
+
+  createLotto = (lottoAmount) => {
+    for (let i = 0; i < lottoAmount; i++) {
+      const lotto = new Lotto(Random.getnerateRandomNumbers());
+      this.#lottoArray.push(lotto);
+    }
+  };
+
+  printLotto = (lottoAmount) => {
+    $(".purchase-amount").innerHTML = `총 ${lottoAmount}개를 구매하였습니다.`;
+
+    const lottoList = this.#lottoArray
+      .map((lotto) => {
+        lotto.sortLottoNumbers();
+        return `<li>🎟️ ${lotto.getLottoNumbers()}</li>`;
+      })
+      .join("");
+
+    $(".print-lottos-list").innerHTML = lottoList;
+  };
+
+  getWinningNumbers = () => {
+    try {
+      for (let i = 0; i < SETTINGS.MAX_WINNING_NUMBER_LENGTH; i++) {
+        if ($$(".winning-number")[i].value === "") {
+          throw new Error(ERROR_MESSAGE.EMPTY_INPUT);
+        }
+        this.#winningLotto.push(parseInt($$(".winning-number")[i].value, 10));
+      }
+
+      this.validateWinningNumbers(this.#winningLotto);
+      return this.#winningLotto;
+    } catch (e) {
+      alert(e.message);
+      this.#winningLotto = [];
+    }
+  };
+
+  validateWinningNumbers = (winningLottoNumbers) => {
+    Validator.isDuplicatedNumber(winningLottoNumbers);
+    Validator.isCorrectLength(winningLottoNumbers);
+    
+    for (let i = 0; i < winningLottoNumbers.length; i++) {
+      this.checkEachNumber(winningLottoNumbers[i]);
+    }
+  };
+
+  checkEachNumber = (eachNumber) => {
+    Validator.isNumber(eachNumber);
+    Validator.isCorrectRange(eachNumber);
+    Validator.isPositiveInteger(eachNumber);
+  };
+
+  getBonusNumber = () => {
+    try {
+      this.#bonusNumber = parseInt($(".bonus-number").value, 10);
+      this.validateBonusNumber();
+      this.checkEachNumber(this.#bonusNumber);
+      return this.#bonusNumber;
+    } catch (e) {
+      alert(e.message);
+      this.#bonusNumber = 0;
+    }
+  };
+
+  validateBonusNumber = () => {
+    Validator.hasBonusNumber(this.#bonusNumber, this.#winningLotto);
+  };
+
+  compareLottos = (e) => {
+    e.preventDefault();
+
+    this.getWinningNumbers();
+    this.getBonusNumber();
+    const lottos = new Lottos(this.#lottoArray);
+
+    lottos.getLottos().forEach((lotto) => {
+      lotto.compareNumbers(this.#winningLotto);
+      lotto.checkBonusNumber(this.#bonusNumber);
+    });
+
+    lottos.compareLottosScore();
+    this.printResult(lottos);
+  };
+
+  printResult = (lottos) => {
+    $(".lotto-result-wrap").classList.add("show");
+    $(".three").innerHTML = `${lottos.getLottoRanking()[SCORE.THREE]}개`;
+    $(".four").innerHTML = `${lottos.getLottoRanking()[SCORE.FOUR]}개`;
+    $(".five").innerHTML = `${lottos.getLottoRanking()[SCORE.FIVE]}개`;
+    $(".five-bonus").innerHTML = `${lottos.getLottoRanking()[SCORE.FIVE_BONUS]}개`;
+    $(".six").innerHTML = `${lottos.getLottoRanking()[SCORE.SIX]}개`;
```

Comment:
1. `lottos.getLottoRanking()[잘못된 index 혹은 서버의 데이터]`일 경우 처리는 필요 없을까요?
2. 이것도 어떻게 보면 정해진 N 회 렌더링인데 한번에 그리는게 성능에 더 좋지않을까요?

Reply:
1. `lottos.getLottoRanking()[잘못된 index 혹은 서버의 데이터]`일 경우에는 undefined가 오면 에러처리하는 방법으로 처리할 수 있을까요?

2. 아래의 방법으로 바꿔볼 수 있을 것 같은데 혹시 다른 방법이 있을까요?
```suggestion
    const scoreBoard = {
      ".three": SCORE.THREE,
      ".four": SCORE.FOUR,
      ".five": SCORE.FIVE,
      ".five-bonus": SCORE.FIVE_BONUS,
      ".six": SCORE.SIX,
    };

    for (const [scoreName, score] of Object.entries(scoreBoard)) {
      $(scoreName).innerHTML = `${lottos.getLottoRanking()[score]}개`;
    }
```

Reply:
후후 더 좋은 방법은 미리 `DOM`을 구성해서 놓는거 아닐까요?
5번 그릴꺼 1번 그려서 리얼 DOM에 반영하는거죠